specs/electra/beacon-chain.md

```diff
@@ -1038,11 +1059,17 @@ def verify_and_notify_new_payload(self: ExecutionEngine,
     execution_payload = new_payload_request.execution_payload
     parent_beacon_block_root = new_payload_request.parent_beacon_block_root
     execution_requests_list = get_execution_requests_list(new_payload_request.execution_requests)  # [New in Electra]
+    target_blobs_per_block = MAX_BLOBS_PER_BLOCK // 2  # [New in Electra:EIP7742]
```

Comment:
IMO, `target_blobs_per_block` should be a part of `NewPayloadRequest` as the request object contains all params passed onto EL call

Reply:
I think I agree. I've made this change.